### DISCUSSION TITLE
Chanel owner moderation

### DISCUSSION THREAD

**Discussion by kingter-sutjiadi** [inspircd/inspircd#2128] [Support] [answered]
Hi, just wondering how to make a channel owner persist even when he re-join channel 

Thank you 🙏

**Comment by sadiepowell** [comment, marked as answer]
You have two options:

1. Use a services package like [Anope](https://www.anope.org/) which allows you to register channels and nicknams.
2. Use [the autoop module](https://docs.inspircd.org/4/modules/autoop/) to preserve access entries across joins (you will also need [the permchannels module](https://docs.inspircd.org/4/modules/autoop/) to persist your channels across restarts).

**Reply by kingter-sutjiadi** [reply]
Hi, thank you @SadieCat 

so
the 1st solution i need another instance to deploy anope?
the 2nd solution we just need to turn on the module?

any downside for 2nd solution?

**Reply by sadiepowell** [reply]
You can run Anope on the same server as your IRCd and link over localhost.

The biggest downside for the second is that user hosts may not be stable.

**Reply by kingter-sutjiadi** [reply]
ok thank you @SadieCat 👍
